autogpt_platform/autogpt_libs/autogpt_libs/rate_limit/limiter.py

```diff
@@ -0,0 +1,41 @@
+import time
+from redis import Redis
+from typing import Tuple
+from .config import RATE_LIMIT_SETTINGS
+
+
+class RateLimiter:
+    def __init__(self, redis_host: str = RATE_LIMIT_SETTINGS.redis_host,
+                 redis_port: str = RATE_LIMIT_SETTINGS.redis_port,
+                 redis_password: str = RATE_LIMIT_SETTINGS.redis_password,
+                 requests_per_minute: int = RATE_LIMIT_SETTINGS.requests_per_minute):
```

Comment:
Could you please run the formatter? (CI isn't picking it up because it isn't triggered by changes in `autogpt_libs`.)

Reply:
#8860